### ISSUE TITLE
Add tags for translation status

### ISSUE THREAD

**Issue by Xanardi** [ModOrganizer2/modorganizer#1723]
**Is your feature request related to a problem? Please describe.**
The translation status of the mods is not visible from MO2.

**Describe the solution you'd like**
I would like to visualize the translation status in MO2.

**Describe alternatives you've considered**
I would like to see a translation status tag added to the mod.
For example, "not translated", "tentative translation", and "translation completed".

**Comment by Al12rs** [comment]
How should Mo2 know or determine the translation status? 

Would the final user set the tag manually?

In that case you could consider using the Notes field or instead adding custom categories to describe the statuses and manually assign those.

**Comment by Xanardi** [comment]
I don't think it is possible to automatically identify the translation status in MO2.
I will consider how to use Note.
Can I use Note as a filter target?

**Comment by Al12rs** [comment]
Notes are also filtered when using the search field at the bottom right.
You can also sort the list alphabetically through notes.

Custom categories would work better for filtering as you would get a dedicated filter for each of them on the filter tab on the left of the modlist (there is the button at the bottom left to show them)
